### ISSUE TITLE
Plural "other" does not work as default key

### ISSUE THREAD

**Issue by graykick** [aissat/easy_localization#352]
Hi, Guys. I'm using easy_localization 3.0.0 version.
And In this version, Plural's "other" key does not work as default key.

This is example.

### json
```
 "money_args": {
    "other": "{} has {} dollars"
  }
```

### code
```
final money = plural('money_args', 0, args: ['John', '0']); // output: money_args.zero
```

Is it expected behavior?
If not, It looks side effect of [this commit](https://github.com/aissat/easy_localization/commit/788b9ffececa2ed7a21513ccd0fee902c61ae45c#diff-ccec5c5fa2c4d7bf6bc0e9748a8836c4bc5ff4a8c59461448d14047f6b242617)

**Comment by Overman775** [comment]
@aissat please look this bug

**Comment by kuhnroyal** [comment]
This is kind of important as translations that worked in 2.x don't work anymore in 3.0.0.

**Comment by Fintasys** [comment]
Thanks for working on this bug, it just started to bother me. Hope this will get merged soon 🙇

**Comment by EskAere** [comment]
Hi, I've seen the issue is fixed on 3.0.1. So I'm using it, thanks! 
I'm just curious, why is it not released on pub.dev ? Any reason for that ?

**Comment by bw-flagship** [comment]
Fixed in 3.0.1
